### PR TITLE
feat(locale): update hacker phrases

### PR DESCRIPTION
**Pull request by jamesbuch**
A minor feature update to hacker phrases.

### CODE REVIEW

File: src/modules/hacker/index.ts [hunk 1]
```diff
@@ -73,6 +73,20 @@ export class HackerModule {
     return this.faker.helpers.arrayElement(this.faker.definitions.hacker.verb);
   }
 
+  /**
+   * Returns a random hacker/IT verb, past participle.
+   *
+   * @example
+   * faker.hacker.verbed() // 'copied'
+   *
+   * @since 8.0.2
```
Comment (Shinigami92) [inline src/modules/hacker/index.ts:82]: A new function wont land in a bug-fix release. This should be at minimum 8.1.0.

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Please split the feature update (and related locale additions) into a separate PR.

**Comment by ST-DDT** [comment]
@larasoftsolutions Can you please do the requested changes please?

File: src/locales/base/hacker/abbreviation.ts [hunk 2]
```diff
@@ -1,44 +1,112 @@
...
+  'OSI',
   'PCI',
   'PNG',
+  'POP',
+  'RAID',
   'RAM',
+  'RDBMS',
+  'ROM',
   'RSS',
+  'RTFM',
```
Comment (matthewmayer) [inline src/locales/base/hacker/abbreviation.ts:86]: Let's remove this one as we normally try to avoid profanity

**Review by matthewmayer — CHANGES_REQUESTED** [review]

**Comment by ST-DDT** [comment]
@jamesbuch Could you please apply the requested changes?

**Comment by ST-DDT** [comment]
Closing this PR as stale. If you are still interested in this PR please update it and apply the requested changes, then we can reopen this PR or open a new one.

**Pull request closed without merging**